src/commands/signzone.rs

```diff
@@ -2710,6 +2739,8 @@ mod test {
 
         let zone_file_path = mk_test_data_abs_path_string("test-data/example.rfc8976-simple");
 
+        // TODO: Remove outdated references to -b which the tests are no longer using.
```

Comment:
What about this TODO?

Reply:
This isn't really anything to do with this PR but it is correct that there are comments talking about `-b` from earlier versions of the tests that used `-b` and now don't, so the comments are outdated and should be edited.

Reply:
Yes, I just didn't have time at the time.